packages/expo-doctor/src/checks/DirectPackageInstallCheck.ts

```diff
@@ -56,6 +55,32 @@ export const directPackageInstallCheckItems: DirectPackageInstallCheckItem[] = [
       )}`,
     sdkVersionRange: '>=49.0.0',
   },
+  {
+    packageName: '@expo/prebuild-config',
+    ...baseCheckItem,
+    // This has been true for a while, but I can't predict if removing it will cause issues in past SDK versions
+    sdkVersionRange: '>=53.0.0',
+  },
+  {
+    packageName: 'expo-modules-core',
+    getMessage: () =>
+      `The package "expo-modules-core" should not be installed directly in your project. You should instead use the exported API from the expo package.`,
+    sdkVersionRange: '*',
+  },
+  {
+    packageName: '@expo/config-plugins',
+    getMessage: () =>
+      `The package "@expo/config-plugins" should not be installed directly in your project. You should instead use "expo/config-plugins" which is a sub-export of the expo package.`,
```

Comment:
This is probably the most problematic change. A lot of third-party config plugins still add peer dependencies on `@expo/config-plugins` instead of `expo/config-plugins`. We can't really do much about it, but we could extend the message here with more information? Not sure. For example:

> If you installed "@expo/config-plugins" because of a peer dependency for a native module that contains a config plugin, the maintainer of this config plugin should switch to the "expo/config-plugins" import.